apps/vaporgui/NavigationEventRouter.cpp

```diff
@@ -34,6 +34,7 @@
 #include <qcheckbox.h>
 #include <QTextEdit>
 #include <QScrollArea>
+#include <cfloat>
```

Comment:
What does <cfloat> change in this file?

Reply:
It was being included at some point in the include tree but was no longer so I had to re-add it.